appreader/src/main/java/com/ul/ims/gmdl/reader/activity/NfcEngagementActivity.kt

```diff
@@ -475,6 +473,36 @@ class NfcEngagementActivity : AppCompatActivity() {
 
             }
 
+            @Throws(FormatException::class)
+            private fun getMaxLengthFromCarrierDataRecord(
+                carrierDataRecord: NdefRecord?
+            ): Int {
+
+                if (carrierDataRecord == null || carrierDataRecord.payload == null) {
+                    throw FormatException("Carrier Data Record Payload not present.")
+                }
+                try {
+                    val carrierDataRecordPayload = carrierDataRecord.payload
+
+                    // The carrier data record payload for NFC is structured as follows (Index - Data):
+                    // 0x10,        - mDL NFC Connection Handover Version.
+                    // 0xFF, 0xFF   - Maximum length of command data field supported by mobile device
```

Comment:
This is defined in 18013-5 right? If so, please mention that and where in the document.

Reply:
Added in the comment, to be clear I changed the comment from 0xFF 0xFF to 0xYY 0xYY as the value of this field is sent by the holder app and it can be different from 0xFF 0xFF